packages/slate-editor/src/modules/editor-v4-components/BookmarkCard/BookmarkCard.module.scss

```diff
@@ -128,12 +129,12 @@
     }
 }
 
-.providerIcon {
+.provider-icon {
```

Comment:
To be honest I don't see a point in these renaimings. What's the reason for making our lives harder and the code less readable? 

`styles.providerIcon` vs `styles['provider-icon']` ?

Reply:
To keep consistency in css naming, everywhere in the app we use this approach as i can see

Reply:
> To keep consistency in css naming, everywhere in the app we use this approach as i can see

But it did make sense before CSS modules, now it doesn't.